tests/testSNMetrics.py

```diff
@@ -265,11 +265,11 @@ def testSNSLMetric(self):
         metric = SNSLMetric(night_collapse=night_collapse)
 
         # run the metric
-        nSL = metric.run(data, slicePoint={'nside': 64})
+        nSL = metric.run(data, slicePoint={'nside': 64, 'ra': 0., 'ebv': 0.})
 
         # and the result should be
-
-        nSL_ref = 0.00012650940
+        # Changing the refernce value because we have new coadd and mag limits
```

Comment:
Does this mean if the metric kwargs change, someone will have to update this magic number below?  Should there be at least an indication of how you pick or calculate this number? Or run the metric in a fixed configuration?